build/pion-boost.inc

```diff
@@ -134,25 +134,22 @@ if test "x$enable_tests" == "xno"; then
 	# Display notice if unit tests are disabled
 	AC_MSG_NOTICE([Unit tests are disabled])
 else
-	BOOST_TRY_LIB=unit_test_framework
-	BOOST_TRY_LINK="$BOOST_HOME_DIR/lib/libboost_${BOOST_TRY_LIB}${BOOST_LIB_EXTENSION}.a"
 	# check if static version of lib is available
-	if !(test -r "$BOOST_TRY_LINK"); then
```

Comment:
OK I think what I will do then (making the change as part of my merge branch) is keep the conditional logic, but make it prefer dynamic instead of static..